### PR TITLE
Tests passed

### PR DESCRIPTION
**Pull request by momos1703**
- [DEMO LINK](https://momos1703.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://momos1703.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,89 @@
...
+:not(:first-of-type).nav__item {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  height: 40px;
+  padding: 10px 0;
```
Comment (Andriy-Shymkiv) [inline src/style.css:46]: better to set just `height: 60px` in such case

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,89 @@
...
+  color: var(--blue-color-for-link);
+}
+
+:not(:first-of-type) .nav__link:link {
+  color: black;
+}
+
+:not(:first-of-type) .nav__link:visited {
+  color: black;
+}
```
Comment (Andriy-Shymkiv) [inline src/style.css:57]: try to remove this styles, because imho they didn't change anything

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,89 @@
...
+
+:not(:first-of-type) .nav__link:hover {
+  color: var(--blue-color-for-link);
+}
+
+.nav__link:after {
+  /* stylelint-disable-next-line string-quotes */
+  content: '';
```
Comment (Andriy-Shymkiv) [inline src/style.css:65]: use double quotes to prevent eslint error

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,89 @@
...
+  height: 4px;
+  bottom: 0;
+  background: var(--blue-color-for-link);
+  border-radius: 2px;
+  display: none;
+}
+
+.is-active:after {
+  /* stylelint-disable-next-line string-quotes */
+  content: '';
```
Comment (Andriy-Shymkiv) [inline src/style.css:77]: here the same, use double quotes

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
good job! left comments

File: src/style.css (hunk 3 above)
Comment (momos1703) [reply src/style.css:65]: Ooo, thank you)

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,89 @@
...
   margin: 0;
+  min-width: 1024px;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  /* stylelint-disable-next-line color-hex-case */
+  --blue-color-for-link: #00ACDC;
```
[the remark this answers is not in the document]
Comment (momos1703) [reply src/style.css:13]: I just copied it from Figma, but you're right

File: src/index.html [hunk 6]
```diff
@@ -2,13 +2,46 @@
...
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="img__link"><img src="images/logo.png" alt="logo"></a>
```
[the remark this answers is not in the document]
Comment (momos1703) [reply src/index.html:16]: Ok, thank you.

[the remark this answers is not in the document]
Comment (momos1703) [reply src/index.html:16]: Thanks a lot)

File: src/style.css (hunk 1 above)
Comment (momos1703) [reply src/style.css:46]: Done!

File: src/style.css (hunk 2 above)
Comment (momos1703) [reply src/style.css:57]: You was right! Just add color for .is-active. Done!

Comment (momos1703) [reply src/style.css:57]: Is this constuction the best fo this task?
:not(:first-of-type).nav__item {
  margin-left: 20px;
}

File: src/index.html (hunk 6 above)
[the remark this answers is not in the document]
Comment (momos1703) [reply src/index.html]: What I should to do for add new commit with corrections? I don't see button "Create pull request"

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,81 @@
...
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  --blue-color-for-link: #00acdc;
+}
+
+header {
```
Comment (DarkMistyRoom) [inline src/style.css:15]: use classes instead of tags

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,81 @@
...
+.nav__link:after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  bottom: 0;
+  background: var(--blue-color-for-link);
+  border-radius: 2px;
+  display: none;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:66]: you should do it only for .is-active. Just remove it

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,81 @@
...
+  color: black;
+}
+
+.is-active {
+  color: var(--blue-color-for-link);
+}
+
+:not(:first-of-type) .nav__link:hover {
+  color: var(--blue-color-for-link);
+}
```
Comment (DarkMistyRoom) [inline src/style.css:55]: combine these selectors

File: src/index.html [hunk 10]
```diff
@@ -2,13 +2,48 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="img__link">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav>
```
Comment (DarkMistyRoom) [inline src/index.html:19]: Add empty lines between multiline sibling blocks of HTML

Comment (DarkMistyRoom) [reply src/index.html:19]: fix it everywhere

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,81 @@
...
+  height: 60px;
+  position: relative;
+  color: black;
+}
+
+.is-active {
+  color: var(--blue-color-for-link);
+}
+
+:not(:first-of-type) .nav__link:hover {
```
Comment (DarkMistyRoom) [inline src/style.css:53]: why do you need this complicated selector? You can set this style for all `.nav__link`s
```suggestion
.nav__link:hover {
```

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Let's improve the code a bit

File: src/style.css (hunk 7 above)
Comment (momos1703) [reply src/style.css:15]: Sorry, but there is noted in checklist:
[CODE STYLE] - Don't use simple tag names or specific styles in class names. **Exception - specific semantic tags, like header, nav, footer etc.** Try to describe the content of the tag.

File: src/style.css (hunk 9 above)
Comment (momos1703) [reply src/style.css:55]: Good idea! Thank you)

File: src/index.html (hunk 10 above)
Comment (momos1703) [reply src/index.html:19]: Done, but I don't sure that everything is correct. Please check.

File: src/style.css (hunk 11 above)
Comment (momos1703) [reply src/style.css:53]: It was just continue of similar decission in the previous part of code. My fault. You're right. Corrected.

File: src/index.html [hunk 12]
```diff
@@ -2,13 +2,59 @@
...
+    <header>
+      
+      <a href="#home" class="img__link">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav>
+        <ul class="nav__list">
+          
+          <li class="nav__item">
```
Comment (Andriy-Shymkiv) [inline src/index.html:24]:
```html
 <ul class="nav__list">
    <li class="nav__item">
```

File: src/index.html [hunk 13]
```diff
@@ -2,13 +2,59 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      
+      <a href="#home" class="img__link">
```
Comment (Andriy-Shymkiv) [inline src/index.html:17]:
```html
<header>
  <a href="#home" class="img__link">
```

File: src/style.css (hunk 7 above)
Comment (Andriy-Shymkiv) [reply src/style.css:15]: you can use class `header` for sure, read again carefully:
  'Exception - specific semantic tags, like `header`, nav, footer etc.'

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
left comments

File: src/style.css (hunk 7 above)
Comment (momos1703) [reply src/style.css:15]: Understood. Than you!

File: src/index.html (hunk 12 above)
Comment (momos1703) [reply src/index.html]: Done

**Comment by momos1703** [comment]
@Andriy-Shymkiv @DarkMistyRoom, Hi! Is this pull request in queue for re-review?

**Pull request closed without merging**